src/webui/index.js

```diff
@@ -171,13 +171,6 @@ module.exports = async function (ctx) {
     }
   })
 
-  ipcMain.on('config.get', () => {
-    window.webContents.send('config.changed', {
-      platform: os.platform(),
-      config: store.store
-    })
-  })
-
```

Comment:
do we really want to remove this? Was it unused?

Reply:
Yes, it is unused and removed in #1425